src/components/ADempiere/Form/VPOS/Collection/index.vue

```diff
@@ -54,7 +59,12 @@
                       :currency="pointOfSalesCurrency"
                     />
                     <el-button slot="reference" type="text" style="color: #000000;font-weight: 604!important;font-size: 100%;">
-                      {{ formatPrice(pending, pointOfSalesCurrency.iSOCode) }}
+                      {{
+                        formatPrice({
+                          value: pending,
+                          currencyCode: pointOfSalesCurrency.iSOCode
```

Comment:
Just can see change for source code with the same functionality, please let it without changes